2018-09-20 18:15:29,781 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-20 18:15:29,796 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-20 18:15:29,796 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-20 18:15:29,796 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-20 18:15:31,671 [10] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/20 10:15:23] | Kind: subscription_charged_successfully
2018-09-20 18:15:53,015 [10] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:ea9f6afc-1b24-4434-8aa6-7040342785c8
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-20 18:15:54,125 [10] ERROR SkieerWeb - Process Error.SubscriptionId:62d09bc8-856f-45ea-b0a6-3e217b540395,PaymentNo:qm7bah6h
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-20 18:24:38,234 [29] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/20 10:24:35] | Kind: subscription_charged_successfully
2018-09-20 18:24:38,296 [29] INFO  SkieerWeb - [WebhookNotification]Subscription not found.SubscriptionId:63c0c0da-2170-41bd-870b-4bd128ad725b,Kind:subscription_charged_successfully
2018-09-20 18:36:07,031 [39] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/20 10:36:07] | Kind: subscription_charged_successfully
2018-09-20 18:36:28,109 [39] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:dddb4bf7-d259-4faf-98ce-f5880b683999
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-20 18:36:29,156 [39] ERROR SkieerWeb - Process Error.SubscriptionId:a4600bf8-6c81-488c-b4fa-8722c94c1202,PaymentNo:4gvm06hn
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
2018-09-20 18:45:29,796 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-20 18:45:29,796 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-20 18:45:29,796 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-20 19:15:59,593 [1] INFO  SkieerWeb - RefundJob starting.
2018-09-20 19:15:59,609 [12] INFO  SkieerWeb - RefundJob checking.
2018-09-20 19:15:59,609 [12] INFO  SkieerWeb - IsEnabledRefund = False
2018-09-20 19:15:59,609 [12] INFO  SkieerWeb - RefundJob waiting 30 minutes.
2018-09-20 19:16:01,312 [11] INFO  SkieerWeb - [Braintree Webhook Received 2018/9/20 10:15:23] | Kind: subscription_charged_successfully
2018-09-20 19:16:22,593 [11] ERROR SkieerWeb - GetAvailableCoupon Error. UserId:ea9f6afc-1b24-4434-8aa6-7040342785c8
System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。 101.200.193.134:9002
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetResponse()
   在 Octoparse.Web.HttpHelper.RESTFullRequest(RequestType method, String url, Object bodyContent, String authorization, String contentType)
   在 Octoparse.Web.HttpHelper.Get[T](String url)
   在 Octoparse.Web.Controllers.SubscribeController.GetAvailableCoupon(String userId, PlanType newPlanType, Decimal newPrice, Decimal nowBillingAmount)
2018-09-20 19:16:23,718 [11] ERROR SkieerWeb - Process Error.SubscriptionId:62d09bc8-856f-45ea-b0a6-3e217b540395,PaymentNo:qm7bah6h
System.ServiceModel.EndpointNotFoundException: 没有终结点在侦听可以接受消息的 http://192.168.0.103:8012/ShareService。这通常是由于不正确的地址或者 SOAP 操作导致的。如果存在此情况，请参见 InnerException 以了解详细信息。 ---> System.Net.WebException: 无法连接到远程服务器 ---> System.Net.Sockets.SocketException: 由于目标计算机积极拒绝，无法连接。 192.168.0.103:8012
   在 System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   在 System.Net.ServicePoint.ConnectSocketInternal(Boolean connectFailure, Socket s4, Socket s6, Socket& socket, IPAddress& address, ConnectSocketState state, IAsyncResult asyncResult, Exception& exception)
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Net.HttpWebRequest.GetRequestStream(TransportContext& context)
   在 System.Net.HttpWebRequest.GetRequestStream()
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   --- 内部异常堆栈跟踪的结尾 ---

Server stack trace: 
   在 System.ServiceModel.Channels.HttpOutput.WebRequestHttpOutput.GetOutputStream()
   在 System.ServiceModel.Channels.HttpOutput.Send(TimeSpan timeout)
   在 System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.SendRequest(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   在 System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   在 System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   在 System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   在 System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   在 Octoparse.Web.OctopusShareService.IOctopusShareService.GetAccount(String userId)
   在 Octoparse.Web.Controllers.SubscribeController.BtHandler()
